hs_tracking/management/commands/stats.py

```diff
@@ -156,17 +158,19 @@ def resources_details(self):
                 r.resource_type,
                 r.size,
                 r.raccess.sharing_status,
+                r.user.userprofile.user_type,
+                r.user_id
             ]
             w.writerow([unicode(v).encode("utf-8") for v in values])
 
     def yesterdays_variables(self):
 
         today_start = timezone.datetime.now().replace(
-            hour=0,
-            minute=0,
-            second=0,
-            microsecond=0
-        )
+             hour=0,
+             minute=0,
+             second=0,
+             microsecond=0
+         )
```

Comment:
Nitpicking here but the extra whitespace is weird here. Did the PEP8 test not pick this up?